modules/custom/activity_basics/src/Plugin/ActivityContext/ContentInMyGroupActivityContext.php

```diff
@@ -102,12 +103,12 @@ public function getRecipients(array $data, int $last_id, int $limit): array {
         // It could happen that a notification has been queued but the content
         // has since been deleted. In that case we can find no additional
         // recipients.
-        if ($post === NULL) {
+        if ($entity === NULL) {
           return $recipients;
         }
 
-        $gid = $post->get('field_recipient_group')->getValue();
-        $owner_id = $post->getOwnerId();
+        $gid = $entity->get('field_recipient_group')->getValue();
```

Comment:
It might make sense to first check if the entity has this field.

Reply:
We don't have this check before and everything worked as expected so I don't think that we should add this check now.